src/page-content/navigation/header.json

```
@@ -5,7 +5,7 @@
       "mainMenu": [
         {
           "title": "Ory Cloud",
-          "description": "Start Free, No Credit Card required",
+          "description": "Start Free",
```

```suggestion
          "description": "Start Free",
```

you know my opinion on this.
keep  "No credit card required" 😁